lua/nvim-lsp-installer/servers/jdtls/init.lua

```diff
@@ -38,7 +39,7 @@ return function(name, root_dir)
                 ),
             },
             "-data",
-            workspace_name,
+            workspace_name .. workspace_dir,
```

Comment:
```suggestion
            patch.concat { workspace_name, workspace_dir },
```

Reply:
should this be patch.concat or path.concat?

Reply:
`path.concat`!